apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx

```diff
@@ -282,9 +281,15 @@ const Indictment = () => {
   )
 
   const initialize = useCallback(() => {
-    if (workingCase.indictmentCounts?.length === 0) {
+    const indictmentCounts = workingCase.indictmentCounts || []
+    if (indictmentCounts.length === 0) {
       handleCreateIndictmentCount()
     }
+    else {
+      console.log("initialise indictment")
+      // in case indictment subtypes have been modified in earlier step
+      setDriversLicenseSuspensionRequest(indictmentCounts)
```

Comment:
We also need to update the suspension request fields if the indictment count have changed due to earlier steps...